app/Http/Controllers/Licenses/LicenseCheckinController.php

```diff
@@ -100,13 +95,18 @@ public function store(Request $request, $seatId = null, $backTo = null)
         $licenseSeat->assigned_to = null;
         $licenseSeat->asset_id = null;
         $licenseSeat->notes = $request->input('notes');
+        if (! $licenseSeat->license->reassignable) {
+            $licenseSeat->unreassignable_seat = true;
+            $licenseSeat->notes .= "\n" . trans('admin/licenses/message.checkin.not_reassignable') . '.';
```

Comment:
I'm not sure we need to append this message. I think we can remove it? If we're going to keep it then it should be in the API controller too.

Reply:
added to the API

Reply:
Do we have other places where we push strings into notes? Once saved the string will never be translated so it might be confusing for end users...not saying it's bad to do but I wish there was another way to signify what happened.

Reply:
I feel kinda weird about this one too tbh. Usually we are the entire note, or we leave the note alone, we won't take a user's note and add onto it